### PR TITLE
Test get_analysis_date()

### PR DESCRIPTION
**Pull request by jdblischak**
I converted the examples to tests. From running covr locally, `get_analysis_date()` now has 95.45% coverage

### CODE REVIEW

**Comment by jdblischak** [comment]
https://app.codecov.io/gh/Merck/simtrial/blob/test-get_analysis_date/R%2Fget_analysis_date.R

**Review by nanxstats — APPROVED** [review]
Nice 👍

File: tests/testthat/test-get_analysis_date.R [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+)
+
+test_that("planned_calendar_time", {
+  observed <- get_analysis_date(simulated_data, planned_calendar_time = 24)
+  expect_equal(observed, 24)
+})
+
+test_that("target_event_overall", {
+  observed <- get_analysis_date(simulated_data, target_event_overall = 300)
+  expect_equal(observed, 25.6150577826303)
```
Comment (LittleBeannie) [inline tests/testthat/test-get_analysis_date.R:51]: Could you kindly explain the purpose of the testing done after line 49? The output 25.6150577826303 is an unvalidated result. I understand that you may have performed these tests to verify the data.table transformation. If that's the case, would it be possible to rename the file to something like "test-compare with original function output - get_analysis_date"?

Comment (jdblischak) [reply tests/testthat/test-get_analysis_date.R:51]: Yes, I need tests in order to refactor the code. This is the only way we will know it continues to behave as you originally designed it. How about `test-unvalidated-get_analysis_date.R`?

**Pull request merged**